resources/resources.yaml

```diff
@@ -170,6 +176,218 @@ spec:
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  name: rollups-default
+  namespace: kube-system
+data:
+  rollups: |
+    [
+      {
+        "retention": "medium",
+        "measurement": "cpu/usage_rate",
+        "name": "cpu/usage_rate/medium",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
+            "field": "value",
+            "alias": "value_mean"
+          }
+        ]
+      },
+      {
+        "retention": "medium",
+        "measurement": "memory/usage",
+        "name": "memory/usage/medium",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
+            "field": "value",
+            "alias": "value_mean"
+          }
+        ]
+      },
+      {
+        "retention": "medium",
+        "measurement": "filesystem/usage",
+        "name": "filesystem/usage/medium",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value"
+          }
+        ]
+      },
+      {
+        "retention": "medium",
+        "measurement": "network/tx",
+        "name": "network/tx/medium",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value"
+          }
+        ]
+      },
+      {
+        "retention": "medium",
+        "measurement": "network/tx_rate",
+        "name": "network/tx_rate/medium",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
```

Comment:
Yeah, averaging over an hour will smooth out all spikes... Sometimes it might be interesting to look at "smooth" historical graphs though to notice trends. Think we can keep it or delete?